### DISCUSSION TITLE
Executable not running - no error given - RedHat 6.10

### DISCUSSION THREAD

**Discussion by ngrilli** [idaholab/moose#16579] [Q&A General] [answered]
Dear MOOSE Users,

After a maintainance break of the NSCC cluster (Singapore) I am having the following problem:

It is a RedHat 6.10
I install MOOSE using the manual procedure, so gcc 9.2.0 and mpich 3.3 compiled successfully,
then all the MOOSE installation goes well and MOOSE compiles and links properly.

When I run the executable on an input file:
./moose_test-opt -i input_file.i

The system just gets stuck and nothing happens.
The executable never starts and never finishes.

Because of this, running ./run_tests leads to timeout errors in all the tests.
 
I am repeating the same installation procedure that I was doing two weeks back and MOOSE was working well before.

Also, ldd on the executable gives the right linking.

The problem is both on the front node and on the computational nodes (PBS system)

Other executable (for instance: mpiexec -n 2 echo "hello") works fine on the terminal.

Any idea where to start investigating this issue?
Since I do not get any error message, it's difficult to decide what to try next.

Best Regards,
Nicolò Grilli
National University of Singapore

**Comment by aeslaughter** [comment, marked as answer]
Our primary technician is out until after the new year, so it might take a bit to get the help you need. Did you perform a clean before re-compiling the new version: "make clobberall" or better yet "git clean -fxd"?

**Reply by ngrilli** [reply]
Dear @aeslaughter 

Thank you very much for your suggestions.
I tried but unfortunately my executable is still stuck and not starting.

The only other thing I tried is to compile in debug mode,
then I followed the gdb instructions on your website,
but also executing with gdb does not give useful information and I paste what I get below.

Is there any specific breakpoint I should set to understand what is going on?
I have tried compiling simple hello worlds programs with the same g++ and I can run them.
Hopefully our IT will be able to fix the problem as it appeared after a maintainance break.

[e0546229@nus04 projects]$ gdb --args ./moose/test/moose_test-dbg -i 3d_bar.i
GNU gdb (GDB) Red Hat Enterprise Linux (7.2-92.el6)
Copyright (C) 2010 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-redhat-linux-gnu".
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>...
Reading symbols from /home/users/nus/e0546229/projects/moose/test/moose_test-dbg...done.
(gdb) b MPI_Abort
Function "MPI_Abort" not defined.
Make breakpoint pending on future shared library load? (y or [n]) y
Breakpoint 1 (MPI_Abort) pending.
(gdb) r
Starting program: /home/users/nus/e0546229/projects/moose/test/moose_test-dbg -i 3d_bar.i
[Thread debugging using libthread_db enabled]

**Comment by aeslaughter** [comment]
@milljm Can you help with this?

**Comment by GiudGiud** [comment]
Is this still an issue?
